war/gulpfile.js

```diff
@@ -23,7 +23,7 @@ builder.bundle('src/main/js/pluginSetupWizard.js')
     .withExternalModuleMapping('bootstrap', 'core-assets/bootstrap:bootstrap3', {addDefaultCSS: true})
     .withExternalModuleMapping('handlebars', 'core-assets/handlebars:handlebars3')
     .less('src/main/less/pluginSetupWizard.less')
-    .inDir('src/main/webapp/jsbundles');
+    .inDir('target/classes/org/jenkinsci/setup'); // bundle as an adjunct
```

Comment:
@recena I changed it so that the resources (.js and .css) are loaded from adjunct/classpath locations. This specific changes is what generates the .js bundle and processed css into a classpath location (so the adjunct actually works).